src/schema/v2/userInterests.ts

```diff
@@ -0,0 +1,34 @@
+import { connectionDefinitions } from "graphql-relay"
+import {
+  userInterestInterestUnion,
+  UserInterest,
+  userInterestCategoryEnum,
+} from "./me/userInterests"
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLString,
+  Thunk,
+  GraphQLFieldConfigMap,
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from "graphql"
+import { IDFields } from "./object_identification"
+
+export const edgeFields: Thunk<GraphQLFieldConfigMap<
+  UserInterest,
+  ResolverContext
+>> = () => ({
+  ...IDFields,
+  body: { type: GraphQLString },
+  category: { type: new GraphQLNonNull(userInterestCategoryEnum) },
+  createdByAdmin: {
```

Comment:
This seems to be a relevant piece of info on the edge we'd want to expose from the join model. We don't want to allow editing of interests that weren't created by the admin (but by the user themselves).